### PR TITLE
Layout and transitions cards

### PR DESCRIPTION
**Pull request by mikolaslorant**
## Summary
-Corrected title decorations from previous pr.
-Added transitions on hover over tech components.
-Added text and missing tech components to complete layout (Polymer, Angular, etc.)
-Corrected font to Roboto.
-Layout padding and margin adjustment.
-Missing some text characters because of 404 on trello card link.

## Screenshot
screenshot from 2018-08-10 13-48-44
screenshot from 2018-08-10 13-49-01

## Trello card
-Layout.
-Transitions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
       </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/angular_icon.svg" alt="Angular logo"></img>
+        <h1 class="tech-title">AngularJS</h1>
+        <strong class="tech-text">Release: 2010</strong>
+        <p class="tech-text">AngularJS(commonly referred to as AngularJS 1.X)
+        is a JavaScript framework for building SPAs with an MVC architecture.</p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/vue_icon.svg" alt="VueJS logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 2]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
+        <h1 class="tech-title">VueJS</h1>
+        <strong class="tech-text">Release: 2014</strong>
+        <p class="tech-text">VueJS(pronounced /vju:/, like view) is an open-source progressive
+          JavaScript framework for building user interfaces.</p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/elm_icon.svg" alt="Elm logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 3]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
+        <h1 class="tech-title">Elm</h1>
+        <strong class="tech-text">Release: 2016</strong>
+        <p class="tech-text">Elm is domain-specific functional programming language for
+          declaratively creating web browser-based graphical user interfaces.</p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/polymer_icon.svg" alt="Polymer logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 4]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
+        <strong class="tech-text">Release: 2015</strong>
+        <p class="tech-text">Polymer is an open-source JavaScript library
+          for building web applications using Web Components.</p>
+      </div>
+    </div>
+    <!-- Tech set back-end-->
+    <h1 class="title">Back-End Tools</h1>
+    <div class="tech-set">
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/rails_icon.png" alt="Rails logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 5]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
+        <h1 class="tech-title">Ruby on Rails</h1>
+        <strong class="tech-text">Release: 2005</strong>
+        <p class="tech-text">Ruby on Rails, or Rails, is a server-side web application framework
+          written in Ruby. Rails is an MVC framework providing default structures for a database,
+          a web service, and web pages.</p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/express_icon.svg" alt="NodeJS logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 6]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
+        <h1 class="tech-title">NodeJS</h1>
+        <strong class="tech-text">Release: 2009</strong>
+        <p class="tech-text">Node.js is a JavaScript runtime built on Chrome's V8 JavaScript
+          engine. Uses an event-driven, non blocking I/O model that makes it lightweight
+          and efficient.</p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/aspnet_icon.png" alt="aspnet logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 7]
```diff
@@ -17,8 +17,69 @@ <h1 class="tech-title">React JS</h1>
...
+        <h1 class="tech-title">ASP.NET Core</h1>
+        <strong class="tech-text">Release: 2016</strong>
+        <p class="tech-text">APS.NET Core is the next generation of ASP.NET. It
+          is a modular framework that runs on both the full .NET Framework, on
+          Windows, an the cross-plataform .NET Core.</p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/django_icon.svg" alt="Django logo"></img>
```
Comment (FrankIglesias) [inline index.html]: same

File: scss/titles.scss [hunk 8]
```diff
@@ -2,27 +2,13 @@ $tile-decoration-height: 90px;
...
 
 .title {
+  background: url(assets/images/triangle.svg);
+  background-repeat: no-repeat;
+  background-size: $title-decoration-width $tile-decoration-height;
   color:black;
   font-size: $text-big;
-  margin: 10px;
-  padding-bottom: 35px;
+  margin: 30px 10px 10px 10px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: margin: 30px 10px 10px;

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by mikolaslorant** [comment]
Already Corrected and pushed

File: index.html [hunk 9]
```diff
@@ -17,6 +17,69 @@ <h1 class="tech-title">React JS</h1>
         <p class="tech-text">React is a declarative, component-based
           JavaScript library for building user interfaces.</p>
       </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/images/angular_icon.svg" alt="Angular logo"/>
```
Comment (FrankIglesias) [inline index.html]: add space before slash

File: scss/tech.scss [hunk 10]
```diff
@@ -3,16 +3,16 @@
...
-  width: 190px;
+  width: 192px;
   max-width: 100%;
-  padding: 5px 30px 5px 25px;
+  padding: 5px 25px 2px 16px;
   position: relative;
   text-align: center;
 
-  &::before {
+   &::before {
```
Comment (FrankIglesias) [inline scss/tech.scss]: delete this space

File: scss/tech.scss [hunk 11]
```diff
@@ -21,13 +21,20 @@
     left: -10px;
     top: -10px;
     width: inherit;
-    z-index:-1;
+    z-index: -1;
   }
 }
 
+.tech:hover:before {
+  transition: left 0.5s ease, top 0.5s ease;
```
Comment (FrankIglesias) [inline scss/tech.scss]: nest this hover inside the tech class

Comment (FrankIglesias) [reply scss/tech.scss]: `:hover::before`

File: scss/titles.scss [hunk 12]
```diff
@@ -2,25 +2,13 @@ $tile-decoration-height: 90px;
...
-    height: $tile-decoration-height;
-    margin: 10px 15px;
-    padding-top: 15px;
-    width: $title-decoration-width;
-    position: absolute;
-    top: 0;
-    left: 18px;
-    z-index:-1;
-  }
+  padding-bottom: 30px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: use shorthand for padding

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**